x-pack/plugins/session_view/public/types.ts

```diff
@@ -4,20 +4,16 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { ReactElement, ReactNode } from 'react';
+import { ReactNode } from 'react';
 import { CoreStart } from '../../../../src/core/public';
 import { ProcessEvent, Teletype } from '../common/types/process_tree';
 
 export type SessionViewServices = CoreStart;
 
-export interface SessionViewUIStart {
-  getSessionView: (sessionEntityId: string) => ReactElement;
```

Comment:
I think this type was merged incorrectly, as we want it to have the type of `(props: SessionViewDeps) => JSX.Element`